### PR TITLE
cnf-tests: tuning over bond interfaces should be able to ping

### PR DESCRIPTION
**Pull request by mmirecki**

### CODE REVIEW

File: cnf-tests/testsuites/pkg/networks/network.go [hunk 1]
```diff
@@ -38,16 +38,17 @@ func (b *NetworkAttachmentDefinitionBuilder) WithTuning(sysctls string) *Network
 	return b
 }
 
-func (b *NetworkAttachmentDefinitionBuilder) WithBond(bondName, link1, link2 string) *NetworkAttachmentDefinitionBuilder {
+func (b *NetworkAttachmentDefinitionBuilder) WithBond(bondName, link1, link2 string, mtu int) *NetworkAttachmentDefinitionBuilder {
```
Comment (fedepaol) [inline cnf-tests/testsuites/pkg/networks/network.go:41]: Can you expand the commit for this change (or add a new self contained commit) ?

**Review by SchSeba — COMMENTED** [review]
lgtm just make the small change requested by @fedepaol

**Comment by mmirecki** [comment]
> lgtm just make the small change requested by @fedepaol

Will be part of #1174
I'll rebase this when it's merged

**Pull request merged**